src/wp-content/themes/twentytwentyfour/functions.php

```diff
@@ -147,3 +147,23 @@ function twentytwentyfour_block_styles() {
 endif;
 
 add_action( 'init', 'twentytwentyfour_block_styles' );
+
+if ( ! function_exists( 'twentytwentyfour_img_tag_add_width_and_height_attr' ) ) :
+
+	function twentytwentyfour_img_tag_add_width_and_height_attr( $value, $image, $context, $attachment_id, $image_src ) {
+		$theme_dir = get_template_directory_uri();
+
+		switch ( $image_src ) {
+			// TODO: Add clauses for all other theme bundled images here.
+			case "{$theme_dir}/assets/images/business-hero.webp":
+				return array(
+					'width'  => 2560,
+					'height' => 1200,
+				);
```

Comment:
Would returning a tuple be more WordPressy here?

```suggestion
				return array( 2560, 1200 );
```

It's what `wp_image_src_get_dimensions()` returns, and is similar to `wp_get_attachment_image_src()`.